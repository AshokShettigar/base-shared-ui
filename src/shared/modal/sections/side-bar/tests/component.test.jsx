import React from 'react';
import { shallow } from 'enzyme';

import ModalSideBarSection from '../component';

const defaultProps = {
    children: (<span>content</span>)
};

it('should render component', () => {
    const wrapper = shallow(<ModalSideBarSection {...defaultProps} />);

    expect(wrapper).toMatchSnapshot();
});
