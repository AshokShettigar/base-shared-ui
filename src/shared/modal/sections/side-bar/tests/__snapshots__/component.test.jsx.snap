// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render component 1`] = `
<div
  className="root"
  data-component="ModalSideBarSection"
>
  <span>
    content
  </span>
</div>
`;
